config/horizon.json

```diff
@@ -13,6 +13,7 @@
 	"discover_logged_out_redirect_url": "https://discover.wordpress.com",
 	"rebrand_cities_prefix": "rebrandcitiestestsite",
 	"calendly_jetpack_appointment_url": "https://calendly.com/jetpack-com/jetpack-onboarding-call-test",
+	"difm_typeform_id": "",
```

Comment:
Include this key in horizon as well so that we can expose it to the whole team for testing.

Reply:
This would make it available for our public beta testing group as well. For now, we can use the staging environment, what do you think?